doc/version_management.md

```diff
@@ -0,0 +1,16 @@
+Azure CLI Version Management
+============================
+
+## History
+Azure CLI 2 was used for branding as compared to Azure classic CLI. We intended to keep the MAJOR version fixed at 2 for a while. A regular release only updates the PATCH version, which was a little confusing.
```

Comment:
"A regular release only updates the PATCH version, which was a little confusing."
Maybe we should use same tense.

Reply:
updated